addons/stock/models/stock_move.py

```diff
@@ -604,6 +591,22 @@ def _compute_reservation_date(self):
                     days = move.picking_type_id.reservation_days_before_priority
                 move.reservation_date = fields.Date.to_date(move.date) - timedelta(days=days)
 
+    @api.depends('product_uom', 'move_orig_ids', 'move_dest_ids', 'move_orig_ids.packaging_uom_id', 'move_dest_ids.packaging_uom_id')
+    def _compute_packaging_uom_id(self):
+        for move in self:
+            if move.move_orig_ids.packaging_uom_id:
+                move.packaging_uom_id = move.move_orig_ids[0].packaging_uom_id
```

Comment:
This may be incorrect if multiple move_origs have different `packaging_uom_id` though.